dsl/camel-jbang/camel-jbang-core/src/test/java/org/apache/camel/dsl/jbang/core/commands/ExportTest.java

```diff
@@ -109,6 +110,26 @@ public void shouldExportDifferentVersion(RuntimeType rt) throws Exception {
         }
     }
 
+    @Test
+    public void shouldExportQuarkusCustomParameters() throws Exception {
+        String rhrepo = "https://maven.repository.redhat.com/ga";
+        String rhgroup = "com.redhat.quarkus.platform";
+        String rhver = "3.15.3.SP2-redhat-00001";
```

Comment:
We cannot have unit test that uses vendor specific versions like that, call it something else like -foo or something

Reply:
The export command downloads the artifact from the maven repo, I will mock the catalogmodel.

Reply:
I removed the tests, as it was hard to mock the maven download mechanism. I hope it's ok to move on.

Reply:
ok